e2e/wdio/headless/test.e2e.ts

```diff
@@ -100,15 +100,22 @@ describe('main suite 1', () => {
     it('should be able to handle successive scrollIntoView', async () => {
         await browser.url('http://guinea-pig.webdriver.io')
         await browser.setWindowSize(500, 500)
-        const searchInput = await $('.searchinput')
+        let searchInput = await $('.searchinput')
 
         const scrollAndCheck = async (params?: ScrollIntoViewOptions | boolean) => {
+
+            await browser.refresh()
+            searchInput = await $('.searchinput')
+            await browser.pause(500)
             await searchInput.scrollIntoView(params)
             await browser.pause(500)
             const [wdioX, wdioY] = await browser.execute(() => [
                 window.scrollX, window.scrollY
             ])
 
+            await browser.refresh()
+            searchInput = await $('.searchinput')
```

Comment:
WebdriverIO should be able to re-fetch stale elements

```suggestion
```

Reply:
I have this error : `stale element reference: The element with reference ELEMENT-14 is stale; either the element is no longer attached to the DOM, it is not in the current frame context, or the document has been refreshed`

Reply:
I reverted my changes